### ISSUE TITLE
Import fails if element that never existed on the target system is marked as to delete

### ISSUE THREAD

**Issue by gglnx** [verbb/zen#10]
### Describe the bug

The import fails if an element is marked in the import as to be deleted, but never existed on the target system. The error message:

```
Call to a member function getParent() on null
/vendor/verbb/zen/src/base/Element.php:150
/vendor/verbb/zen/src/services/Import.php:221
```

### Steps to reproduce

1. Install Zen
2. Edit an entry and delete it
3. Try an export and import on another system

### Craft CMS version

4.8.2

### Plugin version

1.0.3@dev

### Multi-site?

Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/zen:"dev-craft-4 as 1.0.3"`.

**Comment by gglnx** [comment]
I did the same fix as you as a quick fix. In the import review, the row is empty, and an error will be displayed if I click on details:

![SCR-20240315-owbh Redacted](https://github.com/verbb/zen/assets/23519/bfeb0aaf-dd80-4533-8468-50e07f8c3214)

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/zen:"dev-craft-4 as 1.0.3"`.

**Comment by engram-design** [comment]
Fixed in [1.0.4](https://github.com/verbb/zen/releases/tag/1.0.4)
